Add BaseColor::mixInHsv()

rv1971 opened the pull request:
Has the same interface as BaseColor::mix(), but interpolation is done in HSV space rather than in RGB space. The result may be closer to what the human eye perceives as "the color in between".

Commit:
Add BaseColor::mixInHsv()

ozdemirburak commented:
Hello, thanks for this awesome contribution @rv1971 . 

Can you please also share the reference formula for this? Because I think I've not seen it before. 

And, adding this method to README.md would be perfect.

Commit:
Add dcoumentation abount BaseColor::mixInHsv()

rv1971 commented:
Thanks. The algorithm is pretty straightforward, so I did not search for a reference. I have added a short explanation to the function. And obviously I've added it to README.
